src/FcsWatch.Core/Types.fs

```diff
@@ -556,7 +556,7 @@ let crackedFsprojBundle useEditFiles builder = MailboxProcessor<CrackedFsprojBun
 [<RequireQualifiedAccess>]
 type WhyCompile =
     | WarmCompile
-    | DetectFileChange
+    | DetectFileChange of fileChanges: FileChange list
```

Comment:
I added this per the issue we were discussing in. #18 
I did not end up using it in my implementation.

Reply:
`DetectFileChange` should be renamed to `DetectFileChanges`

Reply:
If I did not end up using the inner value `fileChanges`, should I keep this change at all?

Reply:
> If I did not end up using the inner value fileChanges, should I keep this change at all?


 
Yes, 


> In addition to sending whether this is a "Run" or "Rerun" just sending the names of the changed files could prove useful.

It's my fault
I misunderstand that you want to send (Origin source files to webhook)

Reply:
Ah yes, sorry I was not clear on wanting the dll names, rather than the source files. 

`cache.EntryFsproj` will get me all built projects? My current approach works, let me know if you see something wrong with it.